app/models/rules_of_origin/rule_set.rb

```diff
@@ -5,7 +5,7 @@
 module RulesOfOrigin
   class RuleSet
     DEFAULT_SOURCE_PATH = Rails.root.join('db/rules_of_origin').freeze
-    DEFAULT_FILE = 'rules_of_origin_210728.csv'
+    DEFAULT_FILE = 'rules_of_origin_210923.csv'
```

Comment:
Maybe this would be easier in an environment variable?

Reply:
Bit split on this - since the files are in the code base, controlling which file to use should probably be controlled within the codebase as well